src/test/java/io/cdap/plugin/servicenow/source/ServiceNowMultiSourceConfigTest.java

```diff
@@ -307,11 +313,14 @@ public void testValidateReferenceName() throws Exception {
     RestAPIResponse restAPIResponse = new RestAPIResponse(httpStatus, headers, responseBody);
     Mockito.when(restApi.executeGet(Mockito.any())).thenReturn(restAPIResponse);
     Mockito.when(restApi.parseResponseToResultListOfMap(restAPIResponse.getResponseBody())).thenReturn(result);
-    serviceNowMultiSourceConfig.validate(mockFailureCollector);
-    Assert.assertEquals(1, mockFailureCollector.getValidationFailures().size());
-    Assert.assertEquals("Invalid reference name 'Reference Name'.",
-                        mockFailureCollector.getValidationFailures().get(0).getMessage());
-
+    try {
+      serviceNowMultiSourceConfig.validate(mockFailureCollector);
+      Assert.fail("Exception is not thrown with valid reference name");
+    } catch (ValidationException e) {
+      Assert.assertEquals(1, mockFailureCollector.getValidationFailures().size());
+      Assert.assertEquals("Invalid reference name 'Reference Name'.",
```

Comment:
testing the error message is brittle and not that helpful. Instead, should test that the correct config property is being set in the ValidationFailure.

Reply:
Replaced the assertion written for error message with the assertion for config property in `ValidationFailure` b450d01ecde6caac6e9e789958aff4c54b3e4cb0